src/resources.rs

```diff
@@ -4,6 +4,8 @@ use kube::CustomResource;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
+pub const APPINSTANCE_CRD_PATH: &str = "config/crd/bases/kubecfg.dev_appinstances.yaml";
```

Comment:
This definitely doesn't scale with more CRDs, but I'm stuck on thinking of a novel way to get "all CRDs in `resources`" :thinking: 

I'll take a :mag: into it